infra/dlv/dlv-setup.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.!/bin/bash
+
+SSH="ssh -o StrictHostKeyChecking=no"
+SCP="scp -q -o StrictHostKeyChecking=no"
+
+function usage() {
+    echo "Usage: $0 -h vch-address" >&2
+    exit 1
+}
+
+while getopts "h" flag
+do
+    case $flag in
+
+        h)
+            # Optional
+            export DLV_TARGET_HOST="$OPTARG"
+            ;;
+
+        *)
+            usage
+            ;;
+    esac
+done
+
+shift $((OPTIND-1))
+
+if [ -z "${DLV_TARGET_HOST}" ]; then
+    usage
+fi
+
+DLV_BIN="$GOPATH/bin/dlv"
+
+if [ -z "${GOPATH}" -o -z "${GOROOT}" ]; then
+    echo GOROOT and GOPATH should be set to point to the current GOLANG enironment
+    exit 1
+fi
+
+# copy dlv binary
+echo -n copying dlv binary..
+if [ -f ${DLV_BIN} ]; then
+    ${SCP} ${DLV_BIN} root@${DLV_TARGET_HOST}:/usr/local/bin
+else
+    echo $DLV_BIN does not exist. Run \"go get github.com/derekparker/delve/cmd/dlv\"
+    exit 1
+fi
+echo done
+
+# copy GOROOT env
+echo -n copying GOROOT environment..
```

Comment:
I believe explicitly setting `GOROOT` is now [actively discouraged](https://dave.cheney.net/2013/06/14/you-dont-need-to-set-goroot-really).

We should use `go env GOROOT`.